packages/flame_network_assets/example/lib/main.dart

```diff
@@ -0,0 +1,43 @@
+import 'dart:async';
+
+import 'package:flame/components.dart';
+import 'package:flame/extensions.dart';
+import 'package:flame/game.dart';
+import 'package:flame/input.dart';
+import 'package:flame_network_assets/flame_network_assets.dart';
+import 'package:flutter/material.dart' hide Image;
+
+void main() {
+  runApp(const GameWidget.controlled(gameFactory: MyGame.new));
+}
+
+class MyGame extends FlameGame with TapDetector {
+  final networkImages = FlameNetworkImages();
+  late Image charSprite;
```

Comment:
```suggestion
  late Image playerSprite;
```
Or something that isn't abbreviated.

Reply:
Never stoped to think that char is an abbreviation...